src/containers/Headers.js

```diff
@@ -21,18 +21,47 @@ const mapDispatchToProps = dispatch => bindActionCreators({
 
 const hasClass = (className, classString) => classString.match(className) !== null
 
-const updateFilterSort = (id, isAsc, sorts) => [{id, isAsc}]
+const updateFilterSort = (id, isAsc) => [{id, isAsc}]
 
 const handlers = {
   updateSort: props => event => {
     const isAsc = !hasClass('sorted-asc', event.target.className)
     const id = event.target.getAttribute('data-id')
-    props.refreshData(props.name, {...props.filter, sorts: updateFilterSort(id, isAsc, props.filter.sorts)})
+    props.refreshData(props.name, {...props.filter, sorts: updateFilterSort(id, isAsc)})
+  }
+}
+
+const sort = props => {
+  // multi column sort support not implemented
+  // lowest sort integer column sorts asc
+  let sortRank
+  let sortId
+
+  const filterSort = def => {
+    if (!sortRank && def.sort) {
+      sortRank = def.sort
+      sortId = def.id
+    }
+    if (def.sort && def.sort < sortRank) {
+      sortId = def.id
+    }
+  }
+
+  props.definition.filter(filterSort)
```

Comment:
user sort here instead:

const sort = (a, b) => {
if (a.sort < b.sort) return a
if (a.sort > b.sort) return b
return 0
}
const sorted  = [...props.definition]
sorted.sort(sort)

Reply:
I'll take another look into this, but I initially did a sort similar to this and it resorted the columns as opposed to the data within the columns :/